pandas/core/dtypes/cast.py

```diff
@@ -146,6 +147,30 @@ def maybe_box_datetimelike(value: Scalar, dtype: Optional[Dtype] = None) -> Scal
     return value
 
 
+def maybe_box_native(value: "Union[Series, Scalar]") -> "Union[Series, Scalar]":
```

Comment:
hmm shouldn't this *just* take a Scalar?

Reply:
My bad - for no good reason I had inserted `maybe_box_native` into the `orient="series"` clause of `DataFrame.to_dict` 

Reverted that, and now this takes `Scalar` only